src/stories/cartesian/brush/BrushComponent.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable import/no-default-export */
```

Comment:
Please disable to eslint rule `import/no-default-export` for the stories folder.

Reply:
Include the other two shown above?

Reply:
After validating that their violation is part of the storybook best practice setup, sure. I am not certain.